src/main/java/org/jnode/fs/xfs/XfsRecord.java

```diff
@@ -1,5 +1,8 @@
 package org.jnode.fs.xfs;
 
+import java.util.Arrays;
```

Comment:
Can remove these imports.

Reply:
The suggested change was applied,  thanks